README.md

```diff
@@ -5,11 +5,7 @@
 [![MIT license](https://img.shields.io/badge/license-MIT-green.svg)](#)
 [![Discord](https://discordapp.com/api/guilds/463752820026376202/widget.png)](https://discord.gg/zxSwN8Z)
 
-Fast and robust node.js binary compiler.
-
-**WARNING:** This project was created for [code-server](https://github.com/cdr/code-server) and may provide limited support.
-
-Why was this made? Why not use `pkg` or `nexe`?
+nbin is a in-house binary compiler made initially for code-server but can be used by other projects as well. Unlike contemporaries like `nbin` and `nexe` we support the following that is normally not supported by them:
```

Comment:
i think we should provide more examples, and remove that this was made for `code-server`. nbin has matured quite a bit and i could see it being used for other stuff tbh

Reply:
I would like getting some examples but looks like digging the used by other projects section might be a little bit tricky to get.